### ISSUE TITLE
Failed to install dependencies of OPEA benchmark tool if Python version is higher than 3.10

### ISSUE THREAD

**Issue by joshuayao** [opea-project/GenAIEval#99]
**Reproduce the issue:**
Ensure python is `3.11`.
```shell
pip install -r requirements.txt
``` 
commit id: [71637c0](https://github.com/opea-project/GenAIEval/commit/71637c0cb5eb7e89db6208b5df2e70fb8631ee88)
The supported Python is 3.x in evals/benchmark/README.MD.

**Error:**
```shell
Collecting pyext==0.7 (from bigcode-eval@ git+https://github.com/bigcode-project/bigcode-evaluation-harness.git@e5c2f31625223431d7987f43b70b75b9d26ba118->-r /root/josh/GenAIEval/requirements.txt (line 1))
  Using cached pyext-0.7.tar.gz (7.8 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [9 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-7t4xdpv1/pyext_419332a2abb045ec82db85220394d51e/setup.py", line 6, in <module>
          import pyext
        File "/tmp/pip-install-7t4xdpv1/pyext_419332a2abb045ec82db85220394d51e/pyext.py", line 117, in <module>
          oargspec = inspect.getargspec
                     ^^^^^^^^^^^^^^^^^^
      AttributeError: module 'inspect' has no attribute 'getargspec'. Did you mean: 'getargs'?
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
``` 

**Workaround:**
Downgrade python to `3.10` or lower.

**Comment by RongLei-intel** [comment]
Looks like only 3.10 works now. 3.9 is not supported and I also encountered this issue with python3.11.

**Comment by lkk12014402** [comment]
upgrading bigcode version can fix the issue, see https://github.com/opea-project/GenAIEval/pull/125

**Comment by joshuayao** [comment]
> upgrading bigcode version can fix the issue, see #125

Thanks for the update.

**Comment by joshuayao** [comment]
Python 3.11 works.
